seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -990,6 +990,9 @@
         "You can add images to your website by using the <code>img</code> element, and point to a specific image's URL using the <code>src</code> attribute.",
         "An example of this would be:",
         "<code>&#60img src=\"https://www.your-image-source.com/your-image.jpg\"&#62</code>",
+        "All <code>img</code> elements MUST have an <code>alt</code> attribute. The text inside an <code>alt</code> attribute is used for screen-readers and is displayed if the image fails to load.",
```

Comment:
Instead of capital letters for "must", can these be in `<strong>` or `<em>` tags instead?